umbraco-cloud/upgrades/migrate-from-umbraco-10-to-11.md

```diff
@@ -0,0 +1,161 @@
+---
+description:
+  In this article we show how you can upgrade your Umbraco 10 Cloud project locally to Umbraco 11 and then migrate the project to a new Umbraco 11 project.
+---
+
+# Migrating a Umbraco 10 project to a Umbraco 11 project
```

Comment:
Be aware: GitBook will automatically change the title here, based on what you called it in the SUMMARY file.

Reply:
Got it  👍